tests/cross_functional/ui/fusion_access/test_virtualmachine_lifecycle.py

```
@@ -0,0 +1,247 @@
+        Setup UI session for the test class.
+
+        Args:
+            setup_ui_class_factory: Factory fixture to setup UI session
+        """
+        setup_ui_class_factory()
+        self.page_nav = PageNavigator()
+        self.base_ui = BaseUI()
+        self.vm_ui = VirtualMachineUI()
+
+    @pytest.mark.polarion_id("OCS-8066")
+    def test_create_and_delete_virtualmachine_from_instancetype(self):
```

Ruff (0.15.21)

[warning] Too many statements (128 > 50)

(PLR0915)